VRCFaceTracking/MainStandalone.cs

```diff
@@ -90,15 +101,24 @@ public static void Initialize()
             if (!bindResults.senderSuccess)
                 Logger.Error("Socket failed to bind to sender port, please ensure it's not already in use by another program or specify a different one instead.");
 
-            _relevantParams = UnifiedTrackingData.AllParameters.SelectMany(p => p.GetBase()).Where(param => param.Relevant);
+            _relevantParams = UnifiedTracking.AllParameters_v1.SelectMany(p => p.GetBase()).Where(param => param.Relevant);
+            _relevantParams_v2 = UnifiedTracking.AllParameters_v2.SelectMany(p => p.GetBase()).Where(param => param.Relevant);
 
             ConfigParser.OnConfigLoaded += () =>
             {
-                _relevantParams = UnifiedTrackingData.AllParameters.SelectMany(p => p.GetBase())
+                _relevantParams = UnifiedTracking.AllParameters_v1.SelectMany(p => p.GetBase())
+                    .Where(param => param.Relevant);
+                _relevantParams_v2 = UnifiedTracking.AllParameters_v2.SelectMany(p => p.GetBase())
                     .Where(param => param.Relevant);
-                UnifiedTrackingData.LatestEyeData.ResetThresholds();
-                _relevantParamsCount = _relevantParams.Count();
-                Logger.Msg("Config file parsed successfully! " + _relevantParamsCount + " parameters loaded");
+
+                // Reset calibration on parameter data.
+                UnifiedTracking.Data.ResetCalibration();
+
+                _relevantParamsCount = _relevantParams.Count() + _relevantParams_v2.Count();
+                Logger.Msg("Config file parsed successfully! " + _relevantParamsCount + " parameters loaded.");
+                
+                if (_relevantParams.Count() > 0)
+                    Logger.Warning(_relevantParams.Count() + " legacy parameters loaded. These are undocumented and outdated parameters.");
```

Comment:
I am assuming there is going to be switch somewhere here as looks like this would always say legacy parameters are loaded.

Reply:
All _relevantParams are the original SRanipal parameters (pulled from UnifiedTracking.AllParameters_V1), while _relevantParams_V2 are all pulled from V2. If _relevantParams happens to contain any parameters we would know that the original parameters were loaded in some capacity.